app_usb_aud_xk_216_mc/configs_build.cmake

```diff
@@ -0,0 +1,46 @@
+#
+# Configs that have only had their build process tested
+#
+if(BUILD_TESTED_CONFIGS)
+
+set(APP_COMPILER_FLAGS_1AMi8o2xxxxxx ${SW_USB_AUDIO_FLAGS} -DAUDIO_CLASS=1
+                                                           -DNUM_USB_CHAN_IN_FS=6
```

Comment:
Config name channel count (8) doesn't match the option (6) - this could be an issue with the original Makefile

Reply:
Fixed in both.